[CI] MixedClusterClientYamlTestSuiteIT test {p0=synonyms/60_synonym_rule_get/Synonym set not found} failing

**Build Scans:**
- [elasticsearch-intake #10964 / 8.16.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/heuk3kgzcygdq)
- [elasticsearch-pull-request #35624 / 8.16.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/c4pz6oq7w3ba6)
- [elasticsearch-pull-request #35620 / 8.16.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/zf4r3bveyanki)
- [elasticsearch-pull-request #35610 / 8.16.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/ipgx7zufpsmbq)
- [elasticsearch-pull-request #35613 / 8.16.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/i5gioa2runpca)
- [elasticsearch-pull-request #35593 / 8.16.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/nwl662665pb4a)
- [elasticsearch-pull-request #35612 / 8.16.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/xmitzpvfjt36g)
- [elasticsearch-pull-request #35608 / 8.16.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/ocpaxttw36ace)
- [elasticsearch-pull-request #35607 / 8.16.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/vqsqpifjpk3j2)
- [elasticsearch-pull-request #35604 / 8.16.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/giznady2od24m)

**Reproduction Line:**
```
./gradlew ":qa:mixed-cluster:v8.16.0#mixedClusterTest" -Dtests.class="org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT" -Dtests.method="test {p0=synonyms/60_synonym_rule_get/Synonym rule not found}" -Dtests.seed=51209AC0E98F450C -Dtests.bwc=true -Dtests.locale=sn -Dtests.timezone=SystemV/HST10 -Druntime.java=22
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupInput:(chainingSystem:HIERARCHICAL,controlStyle:twoLine,ignoreParentSettings:(ignoreFilters:!f,ignoreQuery:!f,ignoreTimerange:!f,ignoreValidations:!t),panels:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:task.keyword,grow:!t,id:'0c0c9cb8-ccd2-45c6-9b13-96bac4abc542',searchTechnique:wildcard,selectedOptions:!(),singleSelect:!t,title:'Gradle%20Task',width:medium),grow:!t,order:0,type:optionsListControl,width:small),'144933da-5c1b-4257-a969-7f43455a7901':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:name.keyword,grow:!t,id:'144933da-5c1b-4257-a969-7f43455a7901',searchTechnique:wildcard,selectedOptions:!('test%20%7Bp0%3Dsynonyms%2F60_synonym_rule_get%2FSynonym%20set%20not%20found%7D'),title:Test,width:medium),grow:!t,order:2,type:optionsListControl,width:medium),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:className.keyword,grow:!t,id:'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850',searchTechnique:wildcard,selectedOptions:!('org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT'),title:Suite,width:medium),grow:!t,order:1,type:optionsListControl,width:medium)))))

**Failure Message:**
```
java.lang.AssertionError: Failure at [synonyms/60_synonym_rule_get:16]: expected [2xx] status code but api [cluster.health] returned [408 Request Timeout] [{"cluster_name":"v8.16.0","status":"yellow","timed_out":true,"number_of_nodes":4,"number_of_data_nodes":4,"active_primary_shards":1,"active_shards":2,"relocating_shards":0,"initializing_shards":0,"unassigned_shards":2,"unassigned_primary_shards":0,"delayed_unassigned_shards":0,"number_of_pending_tasks":0,"number_of_in_flight_fetch":0,"task_max_waiting_in_queue_millis":0,"active_shards_percent_as_number":50.0}]
```

**Issue Reasons:**
- [main] 38 failures in test test {p0=synonyms/60_synonym_rule_get/Synonym set not found} (29.7% fail rate in 128 executions)
- [main] 35 failures in step 8.16.0_bwc-snapshots (29.9% fail rate in 117 executions)
- [main] 26 failures in pipeline elasticsearch-pull-request (28.0% fail rate in 93 executions)
- [main] 9 failures in pipeline elasticsearch-intake (36.0% fail rate in 25 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -351,15 +351,6 @@ tests:
 - class: org.elasticsearch.xpack.inference.services.cohere.CohereServiceTests
   method: testInfer_StreamRequest
   issue: https://github.com/elastic/elasticsearch/issues/114385
-- class: org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT
-  method: test {p0=synonyms/60_synonym_rule_get/Synonym set not found}
-  issue: https://github.com/elastic/elasticsearch/issues/114432
-- class: org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT
-  method: test {p0=synonyms/60_synonym_rule_get/Get a synonym rule}
-  issue: https://github.com/elastic/elasticsearch/issues/114443
-- class: org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT
-  method: test {p0=synonyms/60_synonym_rule_get/Synonym rule not found}
-  issue: https://github.com/elastic/elasticsearch/issues/114444
 - class: org.elasticsearch.xpack.inference.InferenceRestIT
   method: test {p0=inference/30_semantic_text_inference/Calculates embeddings using the default ELSER 2 endpoint}
   issue: https://github.com/elastic/elasticsearch/issues/114412

--- a/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/synonyms/60_synonym_rule_get.yml
+++ b/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/synonyms/60_synonym_rule_get.yml
@@ -15,7 +15,8 @@ setup:
               id: "test-id-3"
   - do:
       cluster.health:
-        index: .synonyms-2
+        index: .synonyms
+        timeout: 1m
         wait_for_status: green
 
 ---

